src/sentry/migrations/0167_rm_copied_useroptions.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 1.11.29 on 2021-03-01 20:08
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def delete_duplicate_useroption_rows(apps, schema_editor):
+    """
+    Delete the rows in UserOption that have already been copied over
+    to the NotificationSetting table
+    """
+    UserOption = apps.get_model("sentry", "UserOption")
+
+    for user_option in RangeQuerySetWrapperWithProgressBar(UserOption.objects.all()):
+        if (
+            user_option.key == "workflow:notifications"
+            or user_option.key == "mail:alert"
+            or user_option.key == "deploy-emails"
+        ):
+            user_option.delete()
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn"t be automatically run in
```

Comment:
Looks like this `"` got caught in a find and replace. We should fix it before it gets propagated to another migration via copy-paste.